megamek/src/megamek/common/RoundWeight.java

```diff
@@ -118,7 +107,7 @@ public static double nearestHalfTon(double weight) {
      * @return       The weight in tons, rounded to the closest ton.
      */
     public static double nearestTon(double weight) {
-        return NEAREST_TON.round(weight);
+        return NEAREST_TON.round(weight, null);
```

Comment:
What's the null do?

Reply:
The only one that needs an Entity argument is STANDARD, which will pass off to next half ton or nearest kg depending on the Entity type. If it's null, it defaults to next half ton. The rest ignore it.